m2c/asm_file.py

```diff
@@ -548,6 +554,14 @@ def process_label(label: str, *, kind: LabelKind) -> None:
                         data = parse_incbin(args, options, warnings)
                         if data is not None:
                             asm_file.new_data_bytes(data)
+                    elif directive == ".obj":  # decomp-toolkit label format
+                        parts = line.split()
```

Comment:
can you make use of `args` instead, which is the arguments split by comma? we don't want to make unnecessary assumptions about whitespace here

Reply:
sure